lib/msf/java/rmi/client/jmx/connection/builder.rb

```diff
@@ -0,0 +1,237 @@
+# -*- coding: binary -*-
+
+module Msf
+  module Java
+    module Rmi
+      module Client
+        module Jmx
+          module Connection
+            module Builder
+
+              # Builds an RMI call to javax/management/remote/rmi/RMIConnectionImpl_Stub#getObjectInstance()
+              # used to retrieve an MBean instance
+              #
+              # @param opts [Hash]
```

Comment:
Yard will automatically add documentation for the `opts` argument if you have `@option` tags referencing it.